src/cross_section_measurement/00_pick_bins.py

```diff
@@ -68,7 +68,7 @@ def main():
     s_min = 0.6
     # we also want the statistical error to be larger than 5%
     # this translates (error -= 1/sqrt(N)) to (1/0.05)^2 = 400
-    n_min = 200
+    n_min = 500
```

Comment:
I would change this back to 200.  Whilst it is fairly arbitrary, it's probably better to leave it as the "default" value that's used for most variables.